### ISSUE TITLE
the npm  version 2.7.0 should update @mapbox/node-pre-gyp

### ISSUE THREAD

**Issue by huxfy** [Automattic/node-canvas#1763]
When I'm trying to install canvas@2.7.0 , it throw a deprecate info.

> deprecate node-pre-gyp@^0.15.0 Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future 

I see the source code has already update https://github.com/Automattic/node-canvas/blob/master/package.json#L52 , but the canvas@2.7.0 (https://www.npmjs.com/package/canvas) is not.

**Comment by sindrekjr** [comment]
I second this issue. Looks like it's just a matter of getting another version published with the latest changes.

**Comment by naffydharni006** [comment]
I am also getting the same error, we need a solution to solve this:
```
npm WARN deprecated node-pre-gyp@0.15.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
```

**Comment by zbjornson** [comment]
This was fixed in 2.8.0.

**Comment by andriusign** [comment]
2.8.0 is failing to install https://github.com/Automattic/node-canvas/issues/1862
